src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -469,4 +477,42 @@ private String evaluateMacro(String template) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    private void pullRequestActionable(List<PotentialAction> paList, List<Fact> factsList) {
+        Job job = run.getParent();
+        ItemGroup parent = job.getParent();
+        if (parent instanceof MultiBranchProject) {
+            BranchProjectFactory projectFactory = ((MultiBranchProject) parent).getProjectFactory();
+            if (projectFactory.isProject(job)) {
+                Branch branch = projectFactory.getBranch(job);
+                SCMHead head = branch.getHead();
+
+                if (head instanceof ChangeRequestSCMHead) {
+                    String pronoun = StringUtils.defaultIfBlank(head.getPronoun(), "Change Request");
+                    String viewName = String.format("View %s", pronoun);
+                    String titleName = String.format("%s Title", pronoun);
+                    String authorName = String.format("%s Author", pronoun);
+
+                    ObjectMetadataAction oma = branch.getAction(ObjectMetadataAction.class);
+                    if (oma != null) {
+                        String urlString = oma.getObjectUrl();
+                        PotentialAction viewPRPotentialAction = new PotentialAction(viewName, urlString);
+                        paList.add(viewPRPotentialAction);
+                        factsList.add(new Fact(titleName, oma.getObjectDisplayName()));
+                    }
+                    ContributorMetadataAction cma = branch.getAction(ContributorMetadataAction.class);
+                    if (cma != null) {
+                        String contributor = cma.getContributor();
+                        String contributorDisplayName = cma.getContributorDisplayName();
+                        String author = StringUtils.defaultIfBlank(cma.getContributor(), cma.getContributorDisplayName());
+                        if (StringUtils.isNotBlank(contributor) && StringUtils.isNotBlank(contributorDisplayName))
+                            author = String.format("%s (%s)", cma.getContributor(), cma.getContributorDisplayName());
```

Comment:
Yhinking if both are so valuable as displayName - could you provide screenshot with this?

Reply:
From Github where both are displayed:
![image](https://user-images.githubusercontent.com/1661688/31359713-6ef6defc-ad4b-11e7-967f-d0309045c33a.png)


From Bitbucket.... where only one is available and its the latter... I would prefer username
![image](https://user-images.githubusercontent.com/1661688/31359702-5ff6c2f0-ad4b-11e7-8962-e898b0bea501.png)

Reply:
Also, I already provided plenty of screenshots on #22